src/Umbraco.Tests/Services/ContentServiceTests.cs

```diff
@@ -68,13 +70,17 @@ public void Deleting_Node_With_Invalid_Path()
             }
 
             //now make the data corrupted :/
-
             DatabaseContext.Database.Execute("UPDATE umbracoNode SET path = '-1' WHERE id = @id", new {id = content.Id});
 
-            //re-get
+            // need to clear the caches otherwise the ContentService will just serve is a cached version with the non-updated path from db.
+            ApplicationContext.Current.ApplicationCache.RuntimeCache.ClearAllCache();

```

Comment:
Singleton! Argh :P there is no need for this, it is available as a property on the test just like the service context